eclipse-collections/src/main/java/org/eclipse/collections/impl/list/mutable/MultiReaderFastList.java

```diff
@@ -126,7 +126,7 @@ public static <T> MultiReaderFastList<T> newList(int capacity)
 
     public static <T> MultiReaderFastList<T> newList(Iterable<? extends T> iterable)
     {
-        return new MultiReaderFastList<>(FastList.newList(iterable));
```

Comment:
I think this should be left as a `FastList` as this is a `MultiReaderFastList`.

Reply:
Discarded this change to MultiReaderFastList.